core/src/test/java/tech/pegasys/eth2signer/core/signing/FilecoinAddressTest.java

```diff
@@ -89,6 +90,12 @@ void addressWithInvalidLengthThrowsError() {
         .hasMessage("Invalid Address length");
   }
 
+  @Test
+  void thinkThiIsBad() {
```

Comment:
may be rename method to "decodeShouldNotFail" ?

Reply:
heh - I've removed it as it was too ahrd to justify as a reference test